docs/Gemfile

```diff
@@ -10,6 +10,7 @@ source "https://rubygems.org"
 #gem "jekyll", "~> 4.2.0"
 # This is the default theme for new Jekyll sites. You may change this to anything you like.
 gem "minima", "~> 2.5"
```

Comment:
You can safely remove the `minima` dependency, no?

Reply:
Yup forgot to remove that, will push a commit that removes it